packages/graphback-runtime-mongodb/src/queryBuilder.ts

```diff
@@ -0,0 +1,137 @@
+import {AdvancedFilter} from '@graphback/runtime';
+
+const AND_FIELD = 'and';
+const OR_FIELD = 'or';
+const NOT_FIELD = 'not';
+
+// Operators that work with simple substitution :P
+const operatorMap = {
+    [AND_FIELD]: '$and',
+    [OR_FIELD]: '$or',
+    [NOT_FIELD]: '$nor',
```

Comment:
$nor means NOT OR right? Shouldn't NOT_FIELD map to $not?

I am not too familiar with MongoDB syntax

Reply:
[$not](https://docs.mongodb.com/manual/reference/operator/query/not/) works slightly differently in Mongo as compared to SQL, I cannot give multiple fields or even multiple objects to $not, it only works for single field like so
```
{
  description: {
    $not: {
      $eq: "MongoDB"
    }
  },
  title: {
    $not: {
      $eq: "MongoDB"
    }
  }
}
```
but with $nor, we can do:
```
{
  $nor: [
    {
      title: { $eq: "MongoDB" }
      description: { $eq: "MongoDB" }
    }
  ]
}
```
It all works out because of the way boolean nor operator works :)